static/install

```diff
@@ -22,12 +24,21 @@ else
 fi
 
 echo "Installing MESG..."
+
+# fetch the latest version of mesg.
 link=$(curl -s https://api.github.com/repos/mesg-foundation/core/releases/latest | grep "browser_download_url.*$system.*$machine" | cut -d : -f 2,3 | tr -d \")
 
 path="/usr/local/bin/mesg-core"
 
+# install mesg-core binary.
 sudo curl --progress -L $link -o $path
 sudo chmod +x $path
 
+# install system services.
+git clone git@github.com:mesg-foundation/core.git /tmp/mesg-core
```

Comment:
Any reasons not to use `https://github.com/mesg-foundation/mesg.com.git` I have all my remotes with https and not git and I remember having some issues using git instead of the https version

Reply:
no reasons, you're right. updating it to fetch from _https_ might prevent some possible problems that can be caused by _ssh_

Reply:
done